Jenkinsfile

```diff
@@ -9,6 +9,20 @@ node('node') {
       echo "Current build result: ${currentBuild.result}"
     }
 
+    stage('Run Versions Test') {
```

Comment:
Shouldn't this be part of the public Jenkins not the private one?

Reply:
Correct me if I'm wrong, @stefanbenten and @littleskunk suggested that it would be better to run on our private jenkins so we can have slack alert when it fails and such.

Reply:
Sounds good.